pair.go

```diff
@@ -93,9 +96,13 @@ func (cli *Client) handlePairSuccess(node *waBinary.Node) {
 	jid, _ := pairSuccess.GetChildByTag("device").Attrs["jid"].(types.JID)
 	lid, _ := pairSuccess.GetChildByTag("device").Attrs["lid"].(types.JID)
 	platform, _ := pairSuccess.GetChildByTag("platform").Attrs["name"].(string)
+	clientPropsBytes := pairSuccess.GetChildByTag("client-props").Content.([]byte)
```

Comment:
This shouldn't panic if it's missing

```suggestion
	clientPropsBytes, _ := pairSuccess.GetChildByTag("client-props").Content.([]byte)
```